backend/graminsta/post/services.py

```diff
@@ -1,9 +1,44 @@
-# -*- coding: UTF-8 -*-
-"""services.py"""
+"""
+Service functions for post module
+"""
 
+import datetime
 from django.contrib.auth import get_user_model
 from django.shortcuts import get_object_or_404
-from .models import FollowRelationship
+from .models import Post, FollowRelationship
+
+
+def create_post(publisher_id, description, img, mention_user_ids):
+    """Create a Post in datebase.
+
+    Parameters
+    ----------
+    publisher_name: string
+        name of the publisher
+    description: string
+        description of the post
+    img:  binary data
+        image content of the post
+
+    Returns
+    -------
+    post: a successfully created post object
+    """
+    # Get publisher object with publisher_name
+    publisher = get_user_model().objects.get(pk=publisher_id)
+    # Get the default image
+    post = Post.objects.create(
+        publisher=publisher,
+        description=description,
+        img=img,
+        created_at=datetime.datetime.now(),
```

Comment:
Since `created_at` has `auto_now_add=True`, Django will auto populate this field for you when you create the `Post`, so you don't need to pass this.